docs/source/docs/log.md

```diff
@@ -0,0 +1,59 @@
+# Logging
+
+## DeepReg logging
+
+DeepReg uses `logging` to log the messages. The log level is controlled by the
+environment variable `DEEPREG_LOG_LEVEL`. The levels are given in the table below. The
+default level is "2".
+
+To adjust the logging level, there are two options. Take the training as example,
+
+- You can first define the environment variable, then run the job.
+
+  ```bash
+  export DEEPREG_LOG_LEVEL=1
+  deepreg_train --gpu "" --config_path config/unpaired_labeled_ddf.yaml --exp_name test
+  ```
+
+- You can define the environment variable while running the job.
+
+  ```bash
+  DEEPREG_LOG_LEVEL=1 deepreg_train --gpu "" --config_path config/unpaired_labeled_ddf.yaml --exp_name test
+  ```
+
+| DEEPREG_LOG_LEVEL | Behavior                                                                                   |
+| ----------------- | ------------------------------------------------------------------------------------------ |
+| "0"               | Log all messages, equivalent to `logging.DEBUG`.                                           |
+| "1"               | Log all messages, equivalent to `logging.DEBUG`.                                           |
```

Comment:
it is confusing when the two "Behaviour" are exactly the same between "0" and "1". If indeed the case, emphasize to avoid confusion, by adding something like "the same as above at the user level"....

Reply:
updated, saying 0 is the same as 1. some more lines are added at the top to explain deepreg logging and tf logging